docs/trulens/tracking/instrumentation/llama_index.md

```diff
@@ -0,0 +1,177 @@
+# 🦙 LlamaIndex Integration
```

Comment:
just curious why replace the notebooks with markdown? I thought some jupyter mkdocs extension does that for us

Reply:
Did this for these docs pages because they're not intended to be runnable notebooks anyways, and the extension does it for us but it's still kind of ugly compared to normal markdown. Perhaps there's some changes we can make to the extension to make that nicer in general, I can look into that.